### PR TITLE
Fix extensions tests

### PR DESCRIPTION
**Pull request by lassoan**
Make extension description comparison insensitive to line ending.
Make SVN optional.
Fix expected order of build artifacts submission.

### CODE REVIEW

File: Extensions/CMake/SlicerFunctionGenerateExtensionDescription.cmake [hunk 1]
```diff
@@ -193,7 +193,7 @@ function(slicer_generate_extension_description_test)
     EXTENSION_STATUS ""
     )
   execute_process(
-    COMMAND ${CMAKE_COMMAND} -E compare_files
+    COMMAND ${CMAKE_COMMAND} -E compare_files --ignore-eol
```
Comment (jcfr) [inline Extensions/CMake/SlicerFunctionGenerateExtensionDescription.cmake:196]: Is this option available with the minimum required version of CMake associated with Slicer ?

Comment (lassoan) [reply Extensions/CMake/SlicerFunctionGenerateExtensionDescription.cmake:196]: I assumed that it is, I have not checked it.

**Comment by jcfr** [comment]
If it is not the case, let's just exclude the test or add a comment.

Otherwise, looks good.

**Comment by jcfr** [comment]
As a follow up, I will push a branch completely removing svn support ... I have it as a wip branch on my workstation

**Review by jcfr — DISMISSED** [review]

**Comment by lassoan** [comment]
ignore-eol option was introduced in [CMake-3.14](https://cmake.org/cmake/help/v3.14/manual/cmake.1.html#run-a-command-line-tool)

Required CMake version:
- Windows -> documentation states 3.15.1
- Linux, Mac -> https://github.com/Slicer/Slicer/blob/master/CMakeLists.txt#L1 -> 3.13.4

Is Slicer can be built using CMake 3.13?

**Comment by lassoan** [comment]
@jcfr Thanks for the review. I've added the CMake version check and merged.

**Pull request merged**